client/payment-details/summary/missing-order-notice/index.tsx

```diff
@@ -0,0 +1,145 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
```

Comment:
There are some internal deps listed here, which should be in another block of imports.